### ISSUE TITLE
filtering mode slider inaccessible with screenreaders / keyboard

### ISSUE THREAD

**Issue by mzanm** [uBlockOrigin/uBOL-home#654]
### Prerequisites

- [x] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality by using the _chat_ icon in the popup panel.
- [x] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [x] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [x] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBOL-home/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [x] The issue is not present after disabling uBO Lite in the browser.

### I tried to reproduce the issue when...

- [x] uBO Lite is the only extension.
- [x] using a new, unmodified browser profile.

### Description

Hello. Sorry if I made a mistake. I'm a blind screenreader user primarily using the keyboard, and using the mouse is difficult especially with dragging, which I assume how you operate the slider. On the extension popup using a screenreader (Narrator on windows), I can't seem to adjust the filtering mode the one for basic, optimal, complete, and possiblly no filtering. either by pressing enter or using the arrows. I also experience this issue on ios with the safari extension where with the iOS's screenreader Voiceover It can't be adjusted either by standard double tapping or tap and hold.
Would be possible to offer an accessible alternative like visually hidden radio group or similar? Thanks. Let me know if i missed some detail.

### A specific URL where the issue occurs.

```text
https://www.google.com/
```

### Steps to Reproduce

On Windows 11 using Chrome, using the preinstalled screenreader Narrator (also occurs with NVDA, which is what I primarily use but using Narrator since it's preinstalled).
1. Open the URL and the  extension screen.
2. launch Narrator by pressing ctrl + windows + enter.
3. Navigate the ui with the arrow keys up and down arrow and try pressing enter or space on `filtering mode` or `optimal`. The filtering mode doesn't change.
4. Press narrator + space to disable Narrator's web navigation (scan mode). Now press tab to navigate using the keyboard only. The filtering mode picker is not focusable with the keyboard.

### Expected behavior

Intuitively, The filtering mode changes either by cycling back and forth. or through some other method.

### Actual behavior

The filtering mode didn't change.

### Troubleshooting information

```text
name: uBlock Origin Lite
version: 2026.329.1951
browser: Chrome 146 (win)
filtering:
 default: optimal
permission: all
rules: 25100
rulesets:
 +adguard-spyware-url
 ara-0
 easylist
 easyprivacy
 fra-0
 pgl
 ublock-badware
 ublock-filters
 urlhaus-full
scripting:
 ublock-filters.isolated
 ublock-filters.main
 easyprivacy.isolated
 easyprivacy.main
 ublock-badware.main
 fra-0.isolated
 fra-0.main
 css-specific
 css-procedural
```

**Comment by gorhill** [comment]
You don’t need to drag the slider, you can just tap or click the bars making up the slider.

Maybe adding aria label to identify the bars would help?

**Comment by mzanm** [comment]
Thanks a lot, sorry for the late reply. It's been greatly improved.

I tested on chrome on windows, chrome on mac, and Safari on iOS. On the first two the labels appear and i can activate them to change the filtering mode.
However on Safari on iOS with version 2026.507.2008 of the extension the labels don't appear with voiceover. It's possible the change hasn't reached yet or a label on a div without an aria role is not enough for it to show, I'm not sure.

Additionally, something minor but the interactable slider labels can't be activated with  keyboard focus alone that is, without a screenreader necessarily by moving the cursor with tab. the `tabindex` attribute might help with this.
Much appreciated. Regards.

**Comment by gorhill** [comment]
I plan additional work by using accessibility dev tools in the browser.
